.github/workflows/wheels-linux.yml

```diff
@@ -0,0 +1,37 @@
+name: wheels-linux
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    name: Python Wheels Linux
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2.1.1
+        with:
+          submodules: recursive
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Build wheels
+        env:
+          CIBW_SKIP: pp* cp27* *-manylinux_i686
+          CIBW_ENVIRONMENT: "CFLAGS='-lrt'"
```

Comment:
Any concern that I'm **overwriting** `CFLAGS` rather than **appending** here?

Reply:
Fixed this, and dropped the need for linking against `rt` by adding this to the cmake file:

```cmake
set(BUILD_BENCHMARKS OFF CACHE BOOL "" FORCE)
set(BUILD_FILTERS OFF CACHE BOOL "" FORCE)
set(BUILD_GENERATORS OFF CACHE BOOL "" FORCE)
set(BUILD_TESTING OFF CACHE BOOL "" FORCE)
```

The benchmarking code was the only thing requiring `rt`, and turning off everything else here helps to reduce build times quite a bit.

Reply:
@bnaul This should reduce your build times if you do have to compile yourself.